std_msgs/Header header
geometry_msgs/Pose pose
geometry_msgs/Point rotation_position
sensor_msgs/PointCloud2 obstacles
float64 velocity
int64 status